src/components/TabletApp/RightSide/AdvancedBook/DateSelector.tsx

```diff
@@ -1,97 +1,200 @@
-import React from "react";
-import { Grid, Typography, TextField, MenuItem } from "@mui/material";
+import React, { FC, useEffect, useState } from "react";
+import { Box, InputLabel, MenuItem, TextField } from "@mui/material";
 import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
 import { DatePicker } from "@mui/x-date-pickers/DatePicker";
+import dayjs, { Dayjs } from "dayjs";
+import axios from "axios";
 
-const hours = [
-    { value: 8, label: "08:00" },
-    { value: 9, label: "09:00" },
-    { value: 10, label: "10:00" },
-    { value: 11, label: "11:00" },
-    { value: 12, label: "12:00" },
-    { value: 13, label: "13:00" },
-    { value: 14, label: "14:00" },
-    { value: 15, label: "15:00" },
-    { value: 16, label: "16:00" },
-    { value: 17, label: "17:00" },
-    { value: 18, label: "18:00" },
-];
+// import { getMeetings } from "../../../../api/getRequests";
+
+interface Props {
+    handleMeetingDate: (values: any) => void;
+    handleStartTime: (values: any) => void;
+    handleEndTime: (values: any) => void;
+}
+
+interface SelectHourObject {
+    val: Dayjs;
+    text: string;
+    isdisabled: boolean;
+}
+
+interface MeetingDateObject {
+    startDate: Dayjs;
+    endDate: Dayjs;
+}
+
+const getMeetings = async () => {
+    return axios
+        .get("http://localhost:3001/meetings")
+        .then((response) => response.data);
+};
```

Comment:
Use the getMeetings from the getRequests file instead of decalring it again here, I see you commented the import above